spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnGraphSpec.scala

```diff
@@ -155,4 +173,17 @@ class DnnGraphSpec extends FlatSpec with Matchers {
       t1 should be(t2)
     }
   }
+
+  "DnnGraph skip primitives" should "be correct" in {
+    Engine.setEngineType(MklDnn)
+    val batchSize = 2
+    val inputShape = Array(batchSize, 1, 28, 28)
+    val input = Tensor[Float](inputShape).rand()
+
+    val dnn = model(inputShape).asInstanceOf[DnnGraph]
+    dnn.evaluate()
+    dnn.compile(Phase.InferencePhase)
+
+    dnn.forward(input).toTensor[Float]
```

Comment:
If this test case passed, it means the skip works correctly? Check the variable will be better ?

Reply:
if this test can run correctly. it means some layers without implementing computOutputShape is skipped when running dnn graph. It is not easy to do variable check as some operation is private in dnn graph.